googlemock/include/gmock/gmock-cardinalities.h

```diff
@@ -120,8 +120,16 @@ class GTEST_API_ Cardinality {
   static void DescribeActualCallCountTo(int actual_call_count,
                                         ::std::ostream* os);
 
+#if GTEST_NEED_DLL_DECL
```

Comment:
I think better create separate macros that will be decalred based on your macros `GTEST_NEED_DLL_DECL`

Reply:
I am going to close this PR. If this problem is still important enough @KindDragon and @duxiuxing could collaborate to create a better solution, perhaps based on oldr #862